internal/db/schema/migrations/oss/postgres/28/01_session_create_time_idx.up.sql

```diff
@@ -0,0 +1,3 @@
+begin;
+create index session_create_time on session (create_time, public_id);
```

Comment:
Is this index used for the order by? Could we add a comment here explaining the need for it?

Reply:
hmm i actually intended to add that as a separate commit/PR, i was just writing that up now actually. I can fix it up in this PR though.

Reply:
added a comment